config/routes.rb

```diff
@@ -18,7 +18,9 @@
   delete "/cart", to: "cart#empty"
   delete "/cart/:item_id", to: "cart#remove_item"
 
-  resources :orders, only: [:new, :create, :show]
+  get "/orders/new", to: 'orders#new'
+  get "/orders/:id", to: 'orders#show'
+  post "/orders", to: 'orders#create'
```

Comment:
I'm thinking a refactor could be to resource this one instead of handroll - didn't have time to test

Reply:
I'm good with hand rolling if we don't get around to it.